README.md

```diff
@@ -38,11 +44,12 @@ This action can be used to redeploy application on the DigitalOcean's [App Platf
                       ]'
   ```
 - DigitalOcean App Platform will now update your DOCR information in App Spec and then deploy your application.
-  (Please use unique tag value for DigitalOcean Container Registry Push instead of latest to update DOCR update in App Spec)
+
+**Note: Always use unique tag names to push image to the DigitalOcean Container Registry. This will allow you to deploy your application without delay. [ref](https://docs.digitalocean.com/products/container-registry/quickstart/)**
```

Comment:
Give an example, eg using git short hashes

Reply:
git short hashes is shown in example https://github.com/ParamPatel207/docr_sample. Should I use

- name: Install doctl
  uses: digitalocean/action-doctl@v2
  with:
  token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
- name: Generate GITHUB_SHA
  id: github-sha
  shell: bash
  run: |
    SHORT_SHA=$(echo $GITHUB_SHA | cut -c1-7)
    echo "::set-output name=sha::$SHORT_SHA"
- name: Publish Image to Digital Ocean Container Registry
  shell: bash
  run: |
    doctl registry login --expiry-seconds 600
    docker build . -t registry.digitalocean.com/sample-go/add_sample:${{steps.github-sha.outputs.sha}}
    docker push registry.digitalocean.com/sample-go/add_sample:${{steps.github-sha.outputs.sha}}
   
as an example for short sha and push to docker??????